src/pages/components/variants/inputCheckbox/Disabled.tsx

```diff
@@ -1,16 +1,26 @@
 import { InputCheckbox } from '@boston-scientific/anatomy-react';
+import { Fieldset } from '@boston-scientific/anatomy-react';
 import Example from 'shared/components/Example';
 
 const Disabled = (): JSX.Element => {
   return (
-    <Example>
-      <div className="bsds-form-control">
-        <InputCheckbox label="Disabled checkbox" disabled />
-      </div>
-      <div className="bsds-form-control">
-        <InputCheckbox label="Disabled checked checkbox" disabled defaultChecked />
-      </div>
-    </Example>
+    <>
+      <Example>
```

Comment:
We aren't showing this in SB. Let's chat about whether or not we should add it to SB or remove it from docs. We can log these types of questions in ADS-715